### PR TITLE
Publish solutions for lab 4

### PR DESCRIPTION
**Pull request by sketch873**

### CODE REVIEW

File: labs/lab04/skel/win/1-timer/mytimer.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * SO, 2016 - Lab #4, Signals
+ * SO - Lab #4, Signals
```
Comment (teodutu) [inline labs/lab04/skel/win/1-timer/mytimer.c]:
```suggestion
 * SO
 * Lab #4, Signals
```

File: labs/lab04/sol/win/1-timer/mytimer.c [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/**
+ * SO - Lab #4, Signals
```
Comment (teodutu) [inline labs/lab04/sol/win/1-timer/mytimer.c]:
```suggestion
 * SO
 * Lab #4, Signals
```

File: labs/lab04/sol/win/utils/utils.h [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/**
+ * SO, 2017
```
Comment (teodutu) [inline labs/lab04/sol/win/utils/utils.h]:
```suggestion
 * SO
```

**Review by teodutu — CHANGES_REQUESTED** [review]
Looks good! Apart from the direct code suggestions I made, there are a few more things to fix:

- Use `Signals`, not `Semnale` in the block comments at the beginning of `lab04/skel/lin/utils/utils.h` and remove the newline after this comment.
- Remove the extra newline in `lab04/skel/lin/2-signals/send_normal.sh`.
- Use `Linux`, not `lin` in the block comments at the beginning of Linux files.
- Align the `*`'s in the block comments at the beginning of each file. They should be
```c
/**
 *
 */
```
not
```c
/**
  *
  */
```

**Review by teodutu — DISMISSED** [review]

File: labs/lab04/sol/win/1-timer/mytimer.c [hunk 4]
```diff
@@ -0,0 +1,80 @@
+/**
+ * SO
+ * Lab #4
+ *
+ * Task #1 Solution, Windows
```
Comment (andreia-oca) [inline labs/lab04/sol/win/1-timer/mytimer.c]: Remove `Solution` from here because it's redundant.

File: labs/lab04/sol/win/1-timer/mytimer.c [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+ */
+static void InitSetTimer(void)
+{
+	/*
+	 * TODO
+	 * First, create your timer using CreateWaitableTimer.
+	 * Then set its timeout and its routine using SetWaitableTimer.
+	 * No data needs to be passeed to the completion routine.
+	 * Don't forget to check those calls for errors.
+	 */
```
Comment (andreia-oca) [inline labs/lab04/sol/win/1-timer/mytimer.c]: I think this multi-line comment must be formatted like this:
```
/**
 *
 */
```

**Review by andreia-oca — DISMISSED** [review]
Looks good to me! I just have two nit comments.

**Review by teodutu — APPROVED** [review]

**Review by andreia-oca — APPROVED** [review]

**Pull request merged**